src/Fantomas.Tests/UtilsTests.fs

```diff
@@ -4,45 +4,47 @@ open System
 open NUnit.Framework
 open Fantomas
 open Fantomas.Tests.TestHelper
+open FsCheck
 
-let private mergeAndCompare a b expected =
-    let result =
-        String.merge Environment.NewLine a b
-        |> String.normalizeNewLine
+module String =
```

Comment:
Used a separate module here to distinguish tests against `String` versus those against `List`. Not sure if that's a standard approach - can remove if not.

Reply:
Please remove this, there are no modules in any of the other test files, so this would be a first for this project.